great_expectations/expectations/core/expect_column_min_to_be_between.py

```diff
@@ -219,6 +219,8 @@ class ExpectColumnMinToBeBetween(ColumnAggregateExpectation):
     )
 
     class Config:
+        title = "Expect column minimum to be between"
```

Comment:
I ran this change by our team and they approved it. Made similar changes to `max` -> `maximum`, `stdev` -> `standard deviation`. Since the titles will no longer be class names, they shouldn't use abbreviations.